js_modules/dagster-ui/packages/ui-core/src/app/Util.tsx

```diff
@@ -163,9 +164,9 @@ export const indexedDBAsyncMemoize = <R, U extends (...args: any[]) => Promise<R
 
   const hashToPromise: Record<string, Promise<R>> = {};
 
-  const genHashKey = async (...args: Parameters<U>) => {
+  const genHashKey = weakMapMemoize(async (...args: Parameters<U>) => {
     return hashFn ? hashFn(...args) : hashObject(args);
-  };
+  });
```

Comment:
Don't think it's a big deal but since hashFn is not a parameter to this function it's quietly persisting the generated hashes across `hashFn` changes